src/lib.rs

```diff
@@ -308,6 +308,7 @@ pub fn init_test<WIDGET>(model_param: WIDGET::ModelParam) ->
 {
     gtk::init().map_err(|_| ())?;
     let component = create_widget_test::<WIDGET>(model_param);
+    gtk::WidgetExt::show_all(&component.0.widget().clone().upcast::<gtk::Widget>());
```

Comment:
I'd rather do `.show_all()` instead of using the fully qualified name.

Reply:
It allows to remove one import. :p